WinUIGallery/Helpers/TitleBarHelper.cs

```diff
@@ -6,11 +6,11 @@ namespace WinUIGallery.Helpers;
 
 internal class TitleBarHelper
 {
-    // workaround as Appwindow titlebar doesn't update caption button colors correctly when changed while app is running
+    // workaround as AppWindow TitleBar doesn't update caption button colors correctly when changed while app is running
```

Comment:
I think this work-around may be reviewed when the time is right.

Reply:
Yeah, I believe there are a bunch of fixes for this in 1.7 stable. Keeping this in for now though so we can do the port to `MainWindow` and using the new `TitleBar`.